# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/composite/PaintCounter.cpp
# Commit: 9bcf4283ca00
# Full Hash: 9bcf4283ca003f8572392b8f111361c12eecde3d
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2019-03-31 09:00:52
# Regressor Bug: 1539026
# File Overlap Count: 1
# Description:
#   Bug 1539026 - Skia m74 Moz2D fixes. r=rhunt
# ==============================================================================

diff -r 01b0009b3f67 -r 9bcf4283ca00 gfx/layers/composite/PaintCounter.cpp
--- a/gfx/layers/composite/PaintCounter.cpp	Tue Mar 26 11:06:47 2019 -0400
+++ b/gfx/layers/composite/PaintCounter.cpp	Tue Mar 26 11:06:56 2019 -0400
@@ -14,6 +14,7 @@
 #include "mozilla/Sprintf.h"
 
 #include "mozilla/gfx/HelpersSkia.h"
+#include "skia/include/core/SkFont.h"
 #include "PaintCounter.h"
 
 namespace mozilla {
@@ -48,12 +49,14 @@
                  aCompositeTime.ToMilliseconds());
 
   SkPaint paint;
-  paint.setTextSize(32);
   paint.setColor(SkColorSetRGB(0, 255, 0));
   paint.setAntiAlias(true);
 
+  SkFont font(SkTypeface::MakeDefault(), 32);
+
   mCanvas->clear(SK_ColorTRANSPARENT);
-  mCanvas->drawText(buffer, strlen(buffer), 10, 30, paint);
+  mCanvas->drawSimpleText(buffer, strlen(buffer), kUTF8_SkTextEncoding,
+                          10, 30, font, paint);
   mCanvas->flush();
 
   if (!mTextureSource) {
